FSQCellManifest/FSQCellManifestProtocols.h

```diff
@@ -199,9 +201,9 @@ typedef void (^FSQCellRecordSelectBlock)(NSIndexPath *indexPath, FSQCellManifest
  @note This callback is not sent for views created/dequeued from header/footer records.
  */
 - (void)manifest:(FSQCellManifest *)manifest willConfigureCell:(id)cell
```

Comment:
willConfigureCell is too long so it looks a bit weird:
- (void)manifest:(FSQCellManifest *)manifest
willConfigureCell:(id)cell
       withModel:(id)model
     atIndexPath:(NSIndexPath *)indexPath
          record:(FSQCellRecord *)cellRecord;